### PR TITLE
Some more tidying, and using wget for the binaries

### PR DESCRIPTION
**Pull request by Imraj-Singh**
Using wget from a personal onedrive file instead of storing it within the repository.

### CODE REVIEW

**Comment by DANAJK** [comment]
We also have a Zenodo community for data where you could put items for download if you wish. (Suitable only for stable items and the items remain public forever.) In the Geometry notebooks, there is a get_nifti_data notebook that uses `curl` to access this:
```
# Setup the working directory for the notebook
import notebook_setup
from sirf_exercises import cd_to_working_dir
cd_to_working_dir('Geometry')

import os

# Cet data, unzip
# curl will download into the current directory

!curl -O https://zenodo.org/record/4940072/files/nifti.zip
!unzip nifti.zip
```

**Review by KrisThielemans — CHANGES_REQUESTED** [review]

**Comment by KrisThielemans** [comment]
Relying on `wget` and `curl` command line utilities is generally dangerous as you don't know if they're installed. We use `curl` already (also in `download.sh`), it'll be on the VMs/docker etc, so I'd prefer that.

at some point, we have to replace this with Python download...

File: notebooks/Deep_Learning_PET/.gitattributes [hunk 1]
```diff
@@ -1 +1 @@
-*.torch_model binary
\ No newline at end of file
+*.torch binary
```
[the remark this answers is not in the document]
Comment (Imraj-Singh) [reply notebooks/Deep_Learning_PET/.gitattributes:1]: There are "built-in macro attribute(s)" from my understanding. It's in the "USING MACRO ATTRIBUTES" section. It's equivalent to the text and diff attributes so I though it was fine... I am happy to change, let me know

**Comment by Imraj-Singh** [comment]
> Relying on `wget` and `curl` command line utilities is generally dangerous as you don't know if they're installed. We use `curl` already (also in `download.sh`), it'll be on the VMs/docker etc, so I'd prefer that.
> 
> at some point, we have to replace this with Python download...

I have added some checks to see if the files are downloaded if that helps my case?

**Comment by DANAJK** [comment]
Provided it is clear in the code or comments where the data is located, and where to download it to, I think it is acceptable for people to do it themselves. So, for now it’s fine to provide a OneDrive URL.

I wouldn’t mess with that download_data script at this stage.

**Comment by KrisThielemans** [comment]
> 

agreed!

But @Imraj-Singh please use `curl` unless you're 100% sure that `wget` is available on the 3 systems...

**Comment by Imraj-Singh** [comment]
@DANAJK @KrisThielemans the wget is within one of the notebooks (the 4th), so from my understanding the download will be the same directory as the notebook. So the participant will see the model and results downloaded into the same directory as the notebook which goes onto use it.

@KrisThielemans by 3 systems do you mean windows, linux and mac, or the three servers in jupyter hub (I guess the three servers use different resources). Thanks! I can check this tomorrow rather quickly.

**Comment by KrisThielemans** [comment]
@Imraj-Singh please just replace `wget` with `curl`. Less headache!
(3 systems: VM, docker, jupyterhub, and even if it's there, we probably don't guarantee it)

File: notebooks/Deep_Learning_PET/.gitattributes (hunk 1 above)
[the remark this answers is not in the document]
Comment (KrisThielemans) [reply notebooks/Deep_Learning_PET/.gitattributes:1]: please check. e.g.
 ```
~/devel/SIRF/data/examples/PET$ git check-attr -a Utahscat600k_ca_seg4.s
Utahscat600k_ca_seg4.s: diff: unset
Utahscat600k_ca_seg4.s: text: unset
```

[the remark this answers is not in the document]
Comment (Imraj-Singh) [reply notebooks/Deep_Learning_PET/.gitattributes:1]: ~/SIRF-Exercises/notebooks/Deep_Learning_PET$ git check-attr -a trained.torch
trained.torch: binary: set
trained.torch: diff: unset
trained.torch: merge: unset
trained.torch: text: unset

**Comment by Imraj-Singh** [comment]
@KrisThielemans that's it changed to curl, was rather involved as wget hides the use of cookies and redirect where curl you have to specify. I guess this is living and learning...

File: notebooks/Deep_Learning_PET/.gitattributes (hunk 1 above)
[the remark this answers is not in the document]
Comment (KrisThielemans) [reply notebooks/Deep_Learning_PET/.gitattributes:1]: great

**Comment by KrisThielemans** [comment]
Thanks!

Ready to merge?

**Comment by Imraj-Singh** [comment]
yes please, the codacy changes seem rather trivial unless you'd like those fixed before merging?

**Pull request merged**